CHANGELOG.md

```diff
@@ -1,17 +1,17 @@
 ## 0.76.0 / 2025-08-08
 
-[FEATURE] OAuth2 TLSConfig and ProxyConfig settings. #6735
-[ENHANCEMENT] Use service endpoint role for alerting. #6784
-[FEATURE] Support configuring OAuth2 in ScrapeConfig. #6814
-[FEATURE] Add scale subresource to Alertmanger resource. #6728
-[ENHANCEMENT] set the namespace if empty for alertmanagers to the current namespace in Prometheus CR. #6338
-[ENHANCEMENT] support TLSConfig TLSVersion setting. #6736
-[ENHANCEMENT] optimize `TLSConfig` to `addSafeTLSToYaml`. #6726
+[FEATURE] Add TLS and Proxy settings to OAuth2 configuration for Prometheus and PrometheusAgent CRDs. #6735
+[FEATURE] Add support for OAuth2 in the ScrapeConfig CRD. #6814
+[FEATURE] Add scale subresource to the Alertmanger CRD, allowing Horizontal Pod Autoscalers(HPAs) to be used. #6728
+[FEATURE] Add Scaleway service discovery to the ScrapeConfig CRD. #6711
+[ENHANCEMENT] Make the `namespace` field optional in the Alertmanager endpoints configuration of the Prometheus CRD, if not defined it will use the `default` namespace. #6338
+[ENHANCEMENT] Add support to configure the TLS version for Prometheus, PrometheusAgent and Alertmanager CRDs. #6736
+[ENHANCEMENT] Add API-level validations to Kubernetes SD in the ScrapeConfig CRD. #6678
+[FEATURE] Add `serviceDiscoveryRole` field to the Prometheus and PrometheusAgent CRDs to select between Endpoints (default) and EndpointSlice for discovering scrape and alerting targets. #6672
 [ENHANCEMENT] Add ServiceDiscoveryRole configuration for use of Endpoints vs EndpointSlice. #6672
+[ENHANCEMENT] Add `secret-label-selector` argument to the operator to filter the Secrets being watched. #6731
 [ENHANCEMENT] Add secret label selector to filter secrets to watch. #6731
-[CHANGE] Add listType:=set to Prometheus secrets field. This allows for having a different manager for each secret. #6762
```

Comment:
Hey, sorry if I sounded like I wanted to remove this line from the changelog in the previous review. What I wanted to say is that we need to find a user-friendly log entry.

Looking at [Simon's comment](https://github.com/prometheus-operator/prometheus-operator/pull/6762#pullrequestreview-2191483657), this explains better what the change is, instead of saying `change from atomic to set`.

Could we re-add the entry with the [CHANGE] prefix, but then do one of the two:
1. Improve the wording so it's more user-friendly... or
2. Add a link to Simon's comment, so interested users can read the user-friendly message

Reply:
Wdyt about the new Change?